{"name": "test_add_category[test_003]", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: './images/image_1652590001.PNG'", "trace": "self = <selenium_lesson.web_base.lite_mall.mine_litemall_po.testcase_litemall.test_case_litemall.TestCaseLiteMall object at 0x00000271DAC7F8C8>\ncategory_name = 'c'\n\n    @pytest.mark.parametrize('category_name',('a','b','c','lily','mary','bobo'),\n                             ids=['test_001','test_002','test_003','test_004','test_005','test_006']\n                             )\n    def test_add_category(self,category_name):\n>       res=LoginPage().login().click_category().add_category().create_category(category_name).get_category_result(category_name)\n\ntest_case_litemall.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n..\\page_object\\categorypage.py:27: in get_category_result\n    self.get_screen()\n..\\page_object\\base.py:31: in get_screen\n    attachment_type=allure.attachment_type.PNG)\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\allure_commons\\_allure.py:182: in file\n    plugin_manager.hook.attach_file(source=source, name=name, attachment_type=attachment_type, extension=extension)\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\pluggy\\hooks.py:286: in __call__\n    return self._hookexec(self, self.get_hookimpls(), kwargs)\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\pluggy\\manager.py:93: in _hookexec\n    return self._inner_hookexec(hook, methods, kwargs)\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\pluggy\\manager.py:87: in <lambda>\n    firstresult=hook.spec.opts.get(\"firstresult\") if hook.spec else False,\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\allure_pytest\\listener.py:226: in attach_file\n    self.allure_logger.attach_file(uuid4(), source, name=name, attachment_type=attachment_type, extension=extension)\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\allure_commons\\reporter.py:114: in attach_file\n    plugin_manager.hook.report_attached_file(source=source, file_name=file_name)\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\pluggy\\hooks.py:286: in __call__\n    return self._hookexec(self, self.get_hookimpls(), kwargs)\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\pluggy\\manager.py:93: in _hookexec\n    return self._inner_hookexec(hook, methods, kwargs)\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\pluggy\\manager.py:87: in <lambda>\n    firstresult=hook.spec.opts.get(\"firstresult\") if hook.spec else False,\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\allure_commons\\logger.py:53: in report_attached_file\n    shutil.copy2(source, destination)\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\shutil.py:266: in copy2\n    copyfile(src, dst, follow_symlinks=follow_symlinks)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nsrc = './images/image_1652590001.PNG'\ndst = 'E:\\\\py23\\\\selenium_lesson\\\\web_base\\\\lite_mall\\\\mine_litemall_po\\\\testcase_litemall\\\\report\\\\6be0e461-129c-4217-8675-1b07b183da4e-attachment.png'\n\n    def copyfile(src, dst, *, follow_symlinks=True):\n        \"\"\"Copy data from src to dst.\n    \n        If follow_symlinks is not set and src is a symbolic link, a new\n        symlink will be created instead of copying the file it points to.\n    \n        \"\"\"\n        if _samefile(src, dst):\n            raise SameFileError(\"{!r} and {!r} are the same file\".format(src, dst))\n    \n        for fn in [src, dst]:\n            try:\n                st = os.stat(fn)\n            except OSError:\n                # File most likely does not exist\n                pass\n            else:\n                # XXX What about other special files? (sockets, devices...)\n                if stat.S_ISFIFO(st.st_mode):\n                    raise SpecialFileError(\"`%s` is a named pipe\" % fn)\n    \n        if not follow_symlinks and os.path.islink(src):\n            os.symlink(os.readlink(src), dst)\n        else:\n>           with open(src, 'rb') as fsrc:\nE           FileNotFoundError: [Errno 2] No such file or directory: './images/image_1652590001.PNG'\n\nc:\\users\\longmin\\appdata\\local\\programs\\python\\python37\\lib\\shutil.py:120: FileNotFoundError"}, "attachments": [{"name": "picture", "source": "6be0e461-129c-4217-8675-1b07b183da4e-attachment.png", "type": "image/png"}, {"name": "log", "source": "3ad34f54-a34d-40d8-a864-32f9da1741ce-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "category_name", "value": "'c'"}], "start": 1652589996374, "stop": 1652590001529, "uuid": "10d63fe6-8d77-4d73-8dfa-e63b75c8f68a", "historyId": "352eeb2cbb5a934e09395c5654b24fb0", "testCaseId": "b7281af36e7a3ba241db49a012ce5999", "fullName": "test_case_litemall.TestCaseLiteMall#test_add_category", "labels": [{"name": "suite", "value": "test_case_litemall"}, {"name": "subSuite", "value": "TestCaseLiteMall"}, {"name": "host", "value": "LAPTOP-6GSA86E7"}, {"name": "thread", "value": "11592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case_litemall"}]}